testing/compose_files/docker-compose-multiworker-make.yml

```diff
@@ -0,0 +1,864 @@
+version: '3'
```

Comment:
What's the difference between this and the docker-compose-multiworker.yml?

Reply:
This one uses an env var for the path to volumes. The other one uses a relative path so the cwd must be the same location as that file.
I wasn't sure if there was any testing or tooling already in place, so I just made a new one ending with `-make` for use with the Makefile new CLI param `compose_file`